imgui-binding/src/main/java/imgui/extension/imguizmo/ImGuizmo.java

```diff
@@ -0,0 +1,276 @@
+package imgui.extension.imguizmo;
+
+import imgui.ImDrawList;
+import imgui.ImGui;
+
+public final class ImGuizmo {
+
+    private static float[] matrices = null; //for drawCubes()
```

Comment:
Maybe create array by default and make variable final?

Reply:
No, we don't want unnecessary allocations. If we'd make it final, java would reserve 64 floats, which corresponds to 256 bytes even, if the user does not want to render debugging cubes on the screen.

The allocation should only happen, if the user really wants to render debugging cubes on the screen, hence the allocation in the drawCubes() method.